blocks/social/social.css

```diff
@@ -194,7 +194,7 @@ main .section.social-container a.category-title {
     font-size: var(--heading-font-size-mxl);
   }
 
-  .block.social {
+  main .section.narrow .block.social {
```

Comment:
This implicitly means `social` block will be styled differently when used inside `narrow` section  vs when used in other sections. Just wondering why would we want that, but if there's a reason, then change looks ok to me.

Reply:
Yes, ideally right position fix of social block should only be for narrow